modules/home-environment.nix

```
@@ -338,7 +347,18 @@ in
         //
         (maybeSet "LC_PAPER" cfg.language.paper)
         //
-        (maybeSet "LC_TIME" cfg.language.time);
+        (maybeSet "LC_TIME" cfg.language.time)
+        //
+        (optionalAttrs (hm.os.isLinux cfg.targetOperatingSystem) (
+          let
+            profiles = [ "/nix/var/nix/profiles/default" "$HOME/.nix-profile" ];
+
+            dataDirs = lib.concatStringsSep ":" (map (profile: "${profile}/share") profiles);
```

A little style nitpick (remove the newline separating the two):

```suggestion
            profiles = [ "/nix/var/nix/profiles/default" "$HOME/.nix-profile" ];
            dataDirs = lib.concatStringsSep ":" (map (profile: "${profile}/share") profiles);
```

You don't have to, but it looks better, IMHO.

---

Also, is there a reason you used `"$HOME/.nix-profile"` rather than `"${cfg.profileDirectory}"`?

---

Fixed